src/Entity/CitizenProjectCategory.php

```diff
@@ -21,4 +24,52 @@
  */
 class CitizenProjectCategory extends BaseEventCategory
 {
+    /**
+     * @var CitizenProjectCategorySkill[]|Collection
+     *
+     * @ORM\OneToMany(targetEntity="CitizenProjectCategorySkill", mappedBy="category", indexBy="id", cascade={"all"}, orphanRemoval=true)
+     *
+     * @Assert\Valid
+     */
+    private $categorySkills;
+
+    public function __construct(? string $name = null, ? string $status = parent::DISABLED)
```

Comment:
status is really nullable ?

Reply:
yes, since these entities are instantiated by Sonata with no argument